ColorCoding_Implementation.cpp

```diff
@@ -0,0 +1,35 @@
+#include <string>
+#include <iostream>
+#include "ColorCoding_Implementation.hpp"
+
+TelCoColorCoder::ColorPair TelCoColorCoder::GetColorFromPairNumber(int pairNumber) {
+    int zeroBasedPairNumber = pairNumber - 1;
+    TelCoColorCoder::MajorColor majorColor = 
+        (TelCoColorCoder::MajorColor)(zeroBasedPairNumber / TelCoColorCoder::numberOfMinorColors);
+    TelCoColorCoder::MinorColor minorColor =
+        (TelCoColorCoder::MinorColor)(zeroBasedPairNumber % TelCoColorCoder::numberOfMinorColors);
+    return TelCoColorCoder::ColorPair(majorColor, minorColor);
+}
+
+int TelCoColorCoder::GetPairNumberFromColor(TelCoColorCoder::MajorColor major, TelCoColorCoder::MinorColor minor) {
+    return major * TelCoColorCoder::numberOfMinorColors + minor + 1;
+}
+
+void TelCoColorCoder::Print25PairCodeReferenceManual() 
```

Comment:
The name of the file does not suggest that it contains the reference manual implementation. Think of another name

Reply:
Changed the Naming